pax-examples/rectangle/Cargo.toml

```diff
@@ -9,9 +9,9 @@ repository = "https://www.github.com/pax-lang/pax"
 description = "Example and development app for the Pax monorepo and compiler"
 
 [dependencies]
-pax-lang = { version="0.8.0", path=".pax/pkg/pax-lang" }
-pax-std = { version="0.8.0", path=".pax/pkg/pax-std" }
-pax-compiler = {version = "0.8.0", path=".pax/pkg/pax-compiler", optional = true}
+pax-lang = { version="0.8.0" }
+pax-std = { version="0.8.0" }
+pax-compiler = {version = "0.8.0", optional = true}
 serde_json = {version = "1.0.95", optional = true}
```

Comment:
`pax-cli` and related still need to be modified to automatically point paths to where `.pax/..` is located.